framework/include/base/Attributes.h

```diff
@@ -129,17 +130,21 @@ class AttribVectorTags : public AttribTagBase
 class AttribExecOns : public Attribute
 {
 public:
-  typedef unsigned int Key;
```

Comment:
what s up with unsigned int -> int

Reply:
`unsigned int` was previously wrong. This represents a flag value in a `MultiMooseEnum`, in which the IDs are `int` (we've supported negative values for a long time)